docs/migratation-guide-v1.0.rst

```diff
@@ -99,12 +101,17 @@ Migrating from ``iConn`` to ``Connection`` Constructor
        database: '*LOCAL',
        username: 'myuser',
        password: 'mypassword',
-       url: 'http://myhost:80/',
+       url: 'http://myhost.example.com/cgi-bin/xmlcgi.pgm',
      }
    });
 
+.. _iconn-to-connection-run:
+
 Migrating from ``iConn.run()`` to ``Connection.run()``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You should be able to figure out how to connect using Connection from the docs, however you may not
```

Comment:
That wasn't meant to be in the docs - it was just an aside. Also, I wanted the warning in the above section. If you're already here, you can just read the section. But how would you know you need to read this section if you just skimmed or missed it?

Reply:
I'll remove the added text here. I'll put the warning right after the [differences](https://github.com/IBM/nodejs-itoolkit/pull/209/files#diff-de47a828a9ae0cdd4bd0a2345c97105aR78).

Reply:
I think the warning should go after "Migrating from iConn to Connection Constructor". Also, I think the example code should be split in to two code blocks with separate sections. See my suggestion above.